### DISCUSSION TITLE
What can I do to disable the extra triangle symbol in the snapshot?

### DISCUSSION THREAD

**Discussion by truemanluo** [ohmyzsh/ohmyzsh#12466] [Help] [answered]
OS：macOS Snonma 14.4
snapshot：<img width="154" alt="image" src="https://github.com/ohmyzsh/ohmyzsh/assets/73474937/0ce9d997-1435-426f-bd37-f427a005db34">

I tried to change the theme, but still the same problem. Thanks for help!

**Comment by carlosala** [comment, marked as answer]
That's a feature of iTerm2. See https://github.com/ohmyzsh/ohmyzsh/issues/11637#issuecomment-1659671161

**Reply by truemanluo** [reply]
Thanks. I found the iTerm2's doc(https://iterm2.com/documentation-shell-integration.html) for the reference.
